### PR TITLE
Test Hide country specific fields in the 'Pick or create person' form of the duplicate detection pop-up, in German and French systems

### PR DESCRIPTION
**Pull request by mp-sgent**

### CODE REVIEW

**Comment by mp-sgent** [comment]
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1084/

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java [hunk 1]
```diff
@@ -135,7 +141,34 @@ public CreateNewContactSteps(
...
+        () -> {
+          fillFirstName(samePersonDataContact.getFirstName());
+          fillLastName(samePersonDataContact.getLastName());
+          fillDateOfBirth(samePersonDataContact.getDateOfBirth(), Locale.GERMAN);
+          selectSex(samePersonDataContact.getSex());
+          fillPrimaryPhoneNumber(samePersonDataContact.getPrimaryPhoneNumber());
+          fillPrimaryEmailAddress(samePersonDataContact.getPrimaryEmailAddress());
+          selectReturningTraveler(samePersonDataContact.getReturningTraveler());
+          fillDateOfReport(samePersonDataContact.getReportDate(), Locale.GERMAN);
+          // fillDiseaseOfSourceCase(samePersonDataContact.getDiseaseOfSourceCase());
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java:155]: left commented code

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java [hunk 2]
```diff
@@ -264,6 +297,12 @@ public CreateNewContactSteps(
           webDriverHelpers.waitForPageLoadingSpinnerToDisappear(50);
           webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(UUID_INPUT);
         });
+    When(
+        "^I click on SAVE new contact case button$",
+        () -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(SAVE_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java:304]: maybe addin here to wait after the confirmation popup to appear? or something to make it stable

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java (hunk 2 above)
Comment (mp-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java:304]: There are different possibilities of popups available after that Saving action and I did it without waite here on purpose

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java [hunk 3]
```diff
@@ -135,7 +141,33 @@ public CreateNewContactSteps(
...
+        () -> {
+          fillFirstName(samePersonDataContact.getFirstName());
+          fillLastName(samePersonDataContact.getLastName());
+          fillDateOfBirth(samePersonDataContact.getDateOfBirth(), Locale.GERMAN);
+          selectSex(samePersonDataContact.getSex());
+          fillPrimaryPhoneNumber(samePersonDataContact.getPrimaryPhoneNumber());
+          fillPrimaryEmailAddress(samePersonDataContact.getPrimaryEmailAddress());
+          selectReturningTraveler(samePersonDataContact.getReturningTraveler());
+          fillDateOfReport(samePersonDataContact.getReportDate(), Locale.GERMAN);
+          // fillCaseIdInExternalSystem(samePersonDataContact.getCaseIdInExternalSystem());
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java:155]: remove commented

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java [hunk 4]
```diff
@@ -135,7 +141,33 @@ public CreateNewContactSteps(
...
+          selectMultiDayContact();
+          fillDateOfFirstContact(samePersonDataContact.getDateOfFirstContact(), Locale.GERMAN);
+          fillDateOfLastContact(samePersonDataContact.getDateOfLastContact(), Locale.GERMAN);
+          // fillCaseOrEventInformation(samePersonDataContact.getCaseOrEventInformation());
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java:159]: remove commented

**Review by rdutu-vg — APPROVED** [review]

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**